modules/oe_content_news/config/post_updates/00007_create_fields/field.storage.node.oe_news_sources.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - link
+    - node
+id: node.oe_news_sources
+field_name: oe_news_sources
+entity_type: node
+type: link
+settings: {  }
+module: link
+locked: false
+cardinality: -1
+translatable: true
```

Comment:
We have to indicate the translatability of the field only at the ewcms level.

Reply:
The storage needs to be translatable in order to be able to set the field translatable when needed (ewcms).